app/services/curation_concerns/admin_set_service.rb

```diff
@@ -1,7 +1,11 @@
 module CurationConcerns
   class AdminSetService
-    def initialize(user)
+    attr_reader :user, :context
+    delegate :blacklight_config, :repository, to: :context
+
+    def initialize(user, context)
```

Comment:
I wonder why we need two arguments at all. Presumably the `#current_user` is available on the `context`.